lib/fb/http_request.rb

```diff
@@ -23,6 +23,8 @@ class HTTPRequest
     # @option options [Hash] :params ({}) The params to use as the query
     #   component of the request URI, for instance the Hash +{a: 1, b: 2}+
     #   corresponds to the query parameters +"a=1&b=2"+.
+    # @option options [Integer] :waiting_time The number of seconds
```

Comment:
I was thinking about this more and talked about this whole thing with @mkk-fullscreen. I wonder if we should have the option to turn this off entirely, or have this be a global configuration.

In particular, this could be problematic for any FB requests that occur in the web application, especially since the rate limiting would happen all at once. I think we'd end up DOSing ourselves if enough requests come in to cause all the threads in the server's pool to be sleeping.